Tag update

Moobien opened the pull request:
Removed extraneous sugar tags

Closes #
Closes #

# Changes proposed in this pull request
-
-

## Testing
- [ ] Yes I tested this before submitting it.
- [ ] I also did a multiplayer test.

Review please

Moobien committed:
Tag update

Removed extraneous sugar tags

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/ldtteam/minecolonies?pullRequest=10622) <br/>All committers have signed the CLA.

Moobien committed:
Merge branch 'version/main' into Moobien

Moobien committed:
Missing Building Additions

Added Medieval Oak, Medieval Dark Oak and Medieval Birch nether mines courtesy of FourNation_UK

Moobien committed:
Merge branch 'version/main' into Moobien
